index.js

```diff
@@ -158,34 +166,49 @@ function addContext(url, context) {
 }
 
 async function createVcExamples() {
-  // generate base keypair and signature suite
-  const keyPairEd25519VerificationKey2020 = await Ed25519VerificationKey2020.generate();
-  const keyPairEd25519Multikey = await Ed25519Multikey.from(keyPairEd25519VerificationKey2020);
+  // generate base keypair and signature suites
+  // ecdsa-sd-2023
+  const keyPairEcdsaMultikeyKeyPair = await EcdsaMultikey
+    .generate({curve: 'P-256'});
+  const keyPairEcdsaMultiKey = await EcdsaMultikey
+    .from({...keyPairEcdsaMultikeyKeyPair});
+  const {createSignCryptosuite} = ecdsaSd2023Cryptosuite;
+  const suiteEcdsaMultiKey = new DataIntegrityProof({
+    signer: keyPairEcdsaMultiKey.signer(),
+    cryptosuite: createSignCryptosuite()
```

Comment:
```suggestion
    cryptosuite: createSignCryptosuite({
      mandatoryPointers: ['/issuer']
    })
```

Reply:
The diff here is showing that this change didn't get applied.

Reply:
Weird... GitHub (and I) think it had been...so you may be right that a rebase ate it. Fixing now.

Reply:
Fixed in 8a99111